/usr/local/lib/python3.10/dist-packages/dynamic_rest/routers.py: Error [API]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "FeatureInfoTemplateField()". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.layers.api.serializers.StyleSerializer'>, embed=True, many=False, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.layers.api.serializers.StyleSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.UserSerializer'>, embed=True, many=False, read_only=True, required=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "ContactRoleField('poc', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "ContactRoleField('metadata_author', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "EmbedUrlField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "ThumbnailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleHierarchicalKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleThesaurusKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleRegionSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleTopicCategorySerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.RestrictionCodeTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.LicenseSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SpatialRepresentationTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "DownloadLinkField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetListSerializer]: could not resolve serializer field "FavoriteField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ExtraMetadataSerializer'>, deferred=True, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.layers.api.serializers.AttributeSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "FeatureInfoTemplateField()". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: unable to resolve type hint for function "capabilities_url". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: unable to resolve type hint for function "dataset_ows_url". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.layers.api.serializers.StyleSerializer'>, embed=True, many=False, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.layers.api.serializers.StyleSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ResourceExecutionRequestSerializer'>, deferred=True, embed=False, read_only=True, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.UserSerializer'>, embed=True, many=False, read_only=True, required=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "ContactRoleField('poc', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "ContactRoleField('metadata_author', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "EmbedUrlField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "ThumbnailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleHierarchicalKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleThesaurusKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleRegionSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleTopicCategorySerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.RestrictionCodeTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.LicenseSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SpatialRepresentationTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "DownloadLinkField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetSerializer]: could not resolve serializer field "FavoriteField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/views.py: Warning [DatasetViewSet]: could not resolve "{200, 500}" for PATCH /api/v2/datasets/{dataset_id}/append/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/layers/api/views.py: Warning [DatasetViewSet]: could not resolve "{200}" for PATCH /api/v2/datasets/{dataset_id}/replace/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [DatasetViewSet > SimpleMapSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/serializers.py: Warning [DatasetViewSet > DatasetMetadataSerializer]: could not resolve serializer field "MetadataFileField(required=True)". Defaulting to "string"
/usr/src/geonode/geonode/layers/api/views.py: Warning [DatasetViewSet]: could not resolve "{200}" for PUT /api/v2/datasets/{id}/metadata/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: unable to resolve type hint for function "name". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: unable to resolve type hint for function "href". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: unable to resolve type hint for function "mime_type". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "GeonodeFilePathField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DocumentFieldField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.UserSerializer'>, embed=True, many=False, read_only=True, required=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "ContactRoleField('poc', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "ContactRoleField('metadata_author', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "EmbedUrlField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "ThumbnailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleHierarchicalKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleThesaurusKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleRegionSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleTopicCategorySerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.RestrictionCodeTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.LicenseSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SpatialRepresentationTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DownloadLinkField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "FavoriteField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ResourceExecutionRequestSerializer'>, deferred=True, embed=False, read_only=True, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/documents/api/serializers.py: Warning [DocumentViewSet > DocumentSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ExtraMetadataSerializer'>, deferred=True, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: unable to resolve type hint for function "perms". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.UserSerializer'>, embed=True, many=False, read_only=True, required=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "ContactRoleField('poc', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "ContactRoleField('metadata_author', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleHierarchicalKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleThesaurusKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleRegionSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleTopicCategorySerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.RestrictionCodeTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.LicenseSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SpatialRepresentationTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "EmbedUrlField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "ThumbnailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "DownloadLinkField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [DocumentViewSet > ResourceBaseSerializer]: could not resolve serializer field "FavoriteField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/resource/api/views.py: Warning [ExecutionRequestViewset]: could not derive type of path parameter "id" because it is untyped and obtaining queryset from the viewset failed. Consider adding a type to the path (e.g. <int:id>) or annotating the parameter type with @extend_schema. Defaulting to "string".
/usr/src/geonode/geonode/facets/views.py: Error [list_facets]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
/usr/src/geonode/geonode/facets/views.py: Error [get_facet]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
/usr/src/geonode/geonode/facets/views.py: Warning [get_facet]: could not derive type of path parameter "facet" because it is untyped and obtaining queryset from the viewset failed. Consider adding a type to the path (e.g. <int:facet>) or annotating the parameter type with @extend_schema. Defaulting to "string".
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.UserSerializer'>, embed=True, many=False, read_only=True, required=False)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "ContactRoleField('poc', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "ContactRoleField('metadata_author', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "EmbedUrlField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "ThumbnailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleHierarchicalKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleThesaurusKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleRegionSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleTopicCategorySerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.RestrictionCodeTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.LicenseSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SpatialRepresentationTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DownloadLinkField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "FavoriteField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DataBlobField(<class 'geonode.base.api.serializers.DataBlobSerializer'>, deferred=True, embed=False, many=False, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ResourceExecutionRequestSerializer'>, deferred=True, embed=False, read_only=True, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/geoapps/api/serializers.py: Warning [GeoAppViewSet > GeoAppSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ExtraMetadataSerializer'>, deferred=True, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [GroupViewSet > GroupProfileSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [GroupViewSet > UserSerializer]: could not resolve serializer field "AvatarUrlField(240, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [GroupViewSet > UserSerializer]: unable to resolve type hint for function "perms". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/harvesting/api/serializers.py: Warning [HarvesterViewSet > BriefHarvesterSerializer]: unable to resolve type hint for function "get_links". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/harvesting/api/serializers.py: Warning [HarvesterViewSet > HarvesterSerializer]: unable to resolve type hint for function "get_links". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/harvesting/api/views.py: Error [HarvestableResourceViewSet]: exception raised while getting serializer. Hint: Is get_serializer_class() returning None or is get_queryset() not working without a request? Ignoring the view for now. (Exception: 'harvester_id')
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicFullyEmbedM2MRelationField(<class 'geonode.maps.api.serializers.MapLayerSerializer'>, deferred=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.UserSerializer'>, embed=True, many=False, read_only=True, required=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "ContactRoleField('poc', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "ContactRoleField('metadata_author', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "EmbedUrlField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "ThumbnailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleHierarchicalKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleThesaurusKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleRegionSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleTopicCategorySerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.RestrictionCodeTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.LicenseSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SpatialRepresentationTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DownloadLinkField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "FavoriteField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DataBlobField(<class 'geonode.base.api.serializers.DataBlobSerializer'>, deferred=True, embed=False, many=False, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ResourceExecutionRequestSerializer'>, deferred=True, embed=False, read_only=True, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ExtraMetadataSerializer'>, deferred=True, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/maps/api/serializers.py: Warning [MapViewSet > MapLayerSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.maps.api.serializers.MapLayerDatasetSerializer'>, embed=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [OwnerViewSet > OwnerSerializer]: could not resolve serializer field "AvatarUrlField(240, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [OwnerViewSet > OwnerSerializer]: unable to resolve type hint for function "perms". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/resource/api/views.py: Error [resource_service_execution_status]: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: unable to resolve type hint for function "perms". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.UserSerializer'>, embed=True, many=False, read_only=True, required=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "ContactRoleField('poc', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "ContactRoleField('metadata_author', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleHierarchicalKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleThesaurusKeywordSerializer'>, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleRegionSerializer'>, embed=True, many=True, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SimpleTopicCategorySerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.RestrictionCodeTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.LicenseSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.SpatialRepresentationTypeSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.GroupSerializer'>, embed=True, many=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DetailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "EmbedUrlField(required=False)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DataBlobField(<class 'geonode.base.api.serializers.DataBlobSerializer'>, deferred=True, embed=False, many=False, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ExtraMetadataSerializer'>, deferred=True, embed=False, many=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DynamicRelationField(<class 'geonode.base.api.serializers.ResourceExecutionRequestSerializer'>, deferred=True, embed=False, read_only=True, required=False, source='id')". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "ThumbnailUrlField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "DownloadLinkField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > ResourceBaseSerializer]: could not resolve serializer field "FavoriteField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for PUT /api/v2/resources/{id}/copy/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for DELETE /api/v2/resources/{id}/delete/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/serializers.py: Warning [ResourceBaseViewSet > FavoriteSerializer]: unable to resolve type hint for function "get_resource". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for GET /api/v2/resources/{id}/ratings/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for POST /api/v2/resources/{id}/ratings/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for PUT /api/v2/resources/{id}/set_thumbnail/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for PUT /api/v2/resources/{id}/update/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for POST /api/v2/resources/{resource_id}/set_thumbnail_from_bbox/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for POST /api/v2/resources/create/{resource_type}/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/base/api/views.py: Warning [ResourceBaseViewSet]: could not resolve "{200}" for POST /api/v2/resources/ingest/{resource_type}/. Expected either a serializer or some supported override mechanism. Defaulting to generic free-form object.
/usr/src/geonode/geonode/upload/api/serializers.py: Warning [UploadSizeLimitViewSet > UploadSizeLimitSerializer]: unable to resolve type hint for function "max_size_label". Consider using a type hint or @extend_schema_field. Defaulting to string.
/usr/src/geonode/geonode/upload/api/serializers.py: Warning [UploadViewSet > UploadSerializer]: could not resolve serializer field "ProgressField(read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/upload/api/serializers.py: Warning [UploadViewSet > UploadSerializer]: could not resolve serializer field "ProgressUrlField('resume', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/upload/api/serializers.py: Warning [UploadViewSet > UploadSerializer]: could not resolve serializer field "ProgressUrlField('delete', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/upload/api/serializers.py: Warning [UploadViewSet > UploadSerializer]: could not resolve serializer field "ProgressUrlField('import', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/upload/api/serializers.py: Warning [UploadViewSet > UploadSerializer]: could not resolve serializer field "ProgressUrlField('detail', read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/upload/api/serializers.py: Warning [UploadViewSet > UploadSerializer]: could not resolve serializer field "UploadFileField(read_only=True, source='resource')". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [UserViewSet > UserSerializer]: could not resolve serializer field "AvatarUrlField(240, read_only=True)". Defaulting to "string"
/usr/src/geonode/geonode/base/api/serializers.py: Warning [UserViewSet > UserSerializer]: unable to resolve type hint for function "perms". Consider using a type hint or @extend_schema_field. Defaulting to string.
Warning: enum naming encountered a non-optimally resolvable collision for fields named "status". The same name has been used for multiple choice sets in multiple components. The collision was resolved with "StatusC7bEnum". add an entry to ENUM_NAME_OVERRIDES to fix the naming.
Warning: enum naming encountered a non-optimally resolvable collision for fields named "status". The same name has been used for multiple choice sets in multiple components. The collision was resolved with "Status175Enum". add an entry to ENUM_NAME_OVERRIDES to fix the naming.
Warning: operationId "api_v2_datasets_partial_update" has collisions [('/api/v2/datasets/', 'patch'), ('/api/v2/datasets/{id}/', 'patch')]. resolving with numeral suffixes.
Warning: operationId "api_v2_datasets_retrieve" has collisions [('/api/v2/datasets/{id}/', 'get'), ('/api/v2/datasets/{id}/{field_name}/', 'get')]. resolving with numeral suffixes.
Warning: operationId "api_v2_documents_partial_update" has collisions [('/api/v2/documents/', 'patch'), ('/api/v2/documents/{id}/', 'patch')]. resolving with numeral suffixes.
Warning: operationId "api_v2_documents_retrieve" has collisions [('/api/v2/documents/{id}/', 'get'), ('/api/v2/documents/{id}/{field_name}/', 'get')]. resolving with numeral suffixes.
Warning: operationId "api_v2_facets_retrieve" has collisions [('/api/v2/facets', 'get'), ('/api/v2/facets/{facet}', 'get')]. resolving with numeral suffixes.
Warning: operationId "api_v2_geoapps_partial_update" has collisions [('/api/v2/geoapps/', 'patch'), ('/api/v2/geoapps/{id}/', 'patch')]. resolving with numeral suffixes.
Warning: operationId "api_v2_geoapps_retrieve" has collisions [('/api/v2/geoapps/{id}/', 'get'), ('/api/v2/geoapps/{id}/{field_name}/', 'get')]. resolving with numeral suffixes.
Warning: operationId "api_v2_groups_partial_update" has collisions [('/api/v2/groups/', 'patch'), ('/api/v2/groups/{id}/', 'patch')]. resolving with numeral suffixes.
Warning: operationId "api_v2_groups_destroy" has collisions [('/api/v2/groups/', 'delete'), ('/api/v2/groups/{id}/', 'delete')]. resolving with numeral suffixes.
Warning: operationId "api_v2_management_commands_retrieve" has collisions [('/api/v2/management/commands/', 'get'), ('/api/v2/management/commands/{cmd_name}/', 'get')]. resolving with numeral suffixes.
Warning: operationId "api_v2_management_commands_create" has collisions [('/api/v2/management/commands/', 'post'), ('/api/v2/management/commands/{cmd_name}/', 'post')]. resolving with numeral suffixes.
Warning: operationId "api_v2_maps_partial_update" has collisions [('/api/v2/maps/', 'patch'), ('/api/v2/maps/{id}/', 'patch')]. resolving with numeral suffixes.
Warning: operationId "api_v2_maps_retrieve" has collisions [('/api/v2/maps/{id}/', 'get'), ('/api/v2/maps/{id}/{field_name}/', 'get')]. resolving with numeral suffixes.
Warning: operationId "api_v2_resources_partial_update" has collisions [('/api/v2/resources/', 'patch'), ('/api/v2/resources/{id}/', 'patch')]. resolving with numeral suffixes.
Warning: operationId "api_v2_resources_destroy" has collisions [('/api/v2/resources/', 'delete'), ('/api/v2/resources/{id}/', 'delete')]. resolving with numeral suffixes.
Warning: operationId "api_v2_resources_retrieve" has collisions [('/api/v2/resources/{id}/', 'get'), ('/api/v2/resources/{id}/{field_name}/', 'get')]. resolving with numeral suffixes.
Warning: operationId "api_v2_resources_copy_update" has collisions [('/api/v2/resources/{id}/copy', 'put'), ('/api/v2/resources/{id}/copy/', 'put')]. resolving with numeral suffixes.
Warning: operationId "api_v2_uploads_upload_create" has collisions [('/api/v2/uploads/upload', 'post'), ('/api/v2/uploads/upload/', 'post')]. resolving with numeral suffixes.
Warning: operationId "api_v2_users_partial_update" has collisions [('/api/v2/users/', 'patch'), ('/api/v2/users/{id}/', 'patch')]. resolving with numeral suffixes.
Warning: operationId "api_v2_users_destroy" has collisions [('/api/v2/users/', 'delete'), ('/api/v2/users/{id}/', 'delete')]. resolving with numeral suffixes.
